src/test/java/core/basesyntax/WriteServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import core.basesyntax.exceptions.InvalidDataException;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.WriteService;
+import core.basesyntax.serviceimpl.FileReaderServiceImpl;
+import core.basesyntax.serviceimpl.WriteServiceImpl;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class WriteServiceTest {
+    private static WriteService writeService;
+    private static FileReaderService fileReaderService;
+
+    @BeforeAll
+    static void setUp() {
+        writeService = new WriteServiceImpl();
+        fileReaderService = new FileReaderServiceImpl();

```

Comment:
Don’t use  FileReaderService here. Create a private method that will read your test files